ztor/src/main/kotlin/com/zenmo/ztor/plugins/Databases.kt

```diff
@@ -50,7 +48,7 @@ fun Application.configureDatabases(): Database {
                 return@post
             }
 
-            val surveyId = surveyRepository.save(survey)
+            val surveyId = surveyRepository.save(survey, call.getUserId())
```

Comment:
This check is not valid in this case because an anonymous user may create a Survey through the form.

The anonymous user may not edit, except through a deeplink. There is no access check preventing them to edit, but the user would have to obtain a valid survey UUID.

I will make a new PR to add an access check for editing.

Reply:
Working on #168